### PR TITLE
Improvement - add api endpoints versions

### PR DESCRIPTION
**Pull request by jcn01**
**docs/rest-api/overview.md:**

- Added `Endpoints` section, starting from `v0.5.5` to `v0.8.3`.

**swagger-v0.8.0 and swagger-v0.8.3:**

- Added new endpoints to Account routes.
   `#operation/getHarvestingDetailsFromAccount`: /account/{accountId}/harvesting
   `#operation/getRegisteredHarvestingAccounts`: /harvesters

- Added new query parameters.
   `#operation/getTransactionsByGroup`: transferMosaicId, fromTransferAmount, toTransferAmount, publicKey, firstLevel
   `#operation/searchMosaics`: holding

### CODE REVIEW

File: website/pages/v0.8.3/endpoints.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    return (
+        <div className="separateOnPageNav">
+            <div className="docMainWrapper wrapper">
+                <Container className="mainContainer documentContainer postContainer">
+                    <div className="post">
+                        <link href="../../css/api.css" rel="stylesheet"></link>
+
+                        <div id="redoc"></div>
+                        <script src="https://cdn.jsdelivr.net/npm/redoc@2.0.0-alpha.41/bundles/redoc.standalone.js"></script>
+                        <script src="../../js/v0.8.3.js"></script>
```
Comment (ynewmann) [inline website/pages/v0.8.3/endpoints.js:16]: not really good at React, but I think we can create one `Endpoints` component and pass `src` to it.

If I'm right
```js
<script src="{props.src}"></script>
```

Comment (jcn01) [reply website/pages/v0.8.3/endpoints.js:16]: I have created a `Endpoint` component and put it under `/website/static/js` directory. But when I try to import and use the component, its returning this error:
```
Error: Cannot find module '../../static/js/Endpoint'
```
Not really sure why it's not working

**Comment by ynewmann** [comment]
Have you checked if everything is displayed correctly? If OK - can merge

**Review by ynewmann — APPROVED** [review]

**Comment by jcn01** [comment]
> Have you checked if everything is displayed correctly? If OK - can merge

Yes, everything is displayed correctly. Ok noted

**Pull request merged**